src/test/java/com/exactpro/junit5/ByteBufUtils.java

```diff
@@ -9,19 +9,20 @@
 
 public class ByteBufUtils {
 
-    public static ByteBuf fromHex(String hex) throws IllegalArgumentFormatException {
-            return buffFromBytes(bytesFromHex(hex));
+    public static ByteBuf fromHex(String hex) throws IllegalArgumentException {
```

Comment:
IllegalArgumentException is a RuntimeException. It's not necessary to specify it in throws